_community/google-groups.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Google Groups - Managing List Membership
+layout: text-optimized-width
+
+---
+
+# {{ page.title }}
+
+This documents how to join and leave a Google Group via webpage and email.
+
+If you have any issues following the guides below please reach out to us at [support@osg-htc.org](mailto:support@osg-htc.org) with the email you would like subscribed and the list you would like to subscribe too.
+
+##### Contents
+
+- [Joining a list via webpage](#joining-a-list-via-webpage)
+- [Leaving a list via webpage](#leaving-a-list-via-webpage)
+- [Joining a list via email](#joining-a-list-via-email)
+- [Leaving a list via email](#leaving-a-list-via-email)
+
+## Joining a list via webpage
+
+To join a group via the Google Groups UI you should:
+
+1. Login to google account you wish to subscribe.
+    - If you don't wish to subscribe via Google account see [below](#joining-a-list-via-email).
+1. Navigate to the groups page at ```https://groups.google.com/u/1/a/osg-htc.org/g/[Group Name]```.
+    - For example group **osg-group**: <br> ```https://groups.google.com/u/1/a/osg-htc.org/g/osg-group```.
+1. Click on the button marked "Join Group" (Shown Below).
+
+#### Notes:
+- This method requires the group is open to the public and a Google Associated email address
+
+![Arrow pointing to Join Groups Button in Google Groups UI](../assets/images/Google_Group_Join_UI.png){: .img-fluid }
+
+## Leaving a list via webpage
+
+1. Login to subscribed email account via Google.
+2. Select group from the ["My Groups" page](https://groups.google.com/my-groups).
+3. Click "My membership settings".
+4. Click "Leave Group" button found in the same location as the "Join Group" button above.
+
+## Joining a list via email
+
+To join a Google Group via email, email ```[Group Name]+subscribe@osg-htc.org```.
```

Comment:
Again, de-emphasize the Google Groups aspect. “To join an OSG Consortium mailing list:”
1. Send email to ...
    Note: Your email does not need a subject or body